sdk-docs/reference/npm-libraries.md

```diff
@@ -0,0 +1,59 @@
+| [@trackunit/nx-utils](https://www.npmjs.com/package/@trackunit/nx-utils) | Contains reusable test utils for [Nx](https://nx.dev/). |
+| [@trackunit/rest-iso-feeds](https://www.npmjs.com/package/@trackunit/rest-iso-feeds) | The purpose of this library is to contain an easy access to trackunits REST apis. |
+| [@trackunit/iris-app-oem-api](https://www.npmjs.com/package/@trackunit/iris-app-oem-api) | Used to add custom brand images into the Trackunit Manager. |
+| [@trackunit/i18n-library-translation](https://www.npmjs.com/package/@trackunit/i18n-library-translation) | Provides i18n translation support for Iris apps. |
+| [@trackunit/iris-app-build-utilities](https://www.npmjs.com/package/@trackunit/iris-app-build-utilities) | Contains utilities for building Iris apps. |
+| [@trackunit/iris-app-webpack-plugin](https://www.npmjs.com/package/@trackunit/iris-app-webpack-plugin) | A custom webpack plugin for Iris applications. |
+
+# Tailwind-based Styling Libraries
+The styling libraries save developers a ton of time and make it easy to streamline the look and feel of the UI. All of our stling components depend on the Tailwind framework, so check out their [official documentation](https://tailwindcss.com/) if you are not familiar with it.
+
+| Library | Usage |
+| --- | --- |
+| [@trackunit/css-core](https://www.npmjs.com/package/@trackunit/css-core) | Includes all the Trackunit css base setup and PostCSS & Tailwind configuration. |
+| [@trackunit/css-components](https://www.npmjs.com/package/@trackunit/css-components) | Includes all the css classes for [Trackunits component library](https://developers.trackunit.com/page/ui-components). |
+| [@trackunit/css-component-tokens](https://www.npmjs.com/package/@trackunit/css-component-tokens) | Exposes the Trackunit Component Tokens Plugin. |
+| [@trackunit/css-tailwind](https://www.npmjs.com/package/@trackunit/css-tailwind) | Exposes the Trackunit Tailwind configuration. |
+| [@trackunit/css-tailwind-custom-properties-plugin](https://www.npmjs.com/package/@trackunit/css-tailwind-custom-properties-plugin) | Exposes a plugin used to convert the variables property in a Tailwind configuration to css custom properties. |
+| [@trackunit/tailwind-styled-components](https://www.npmjs.com/package/@trackunit/tailwind-styled-components) | Is a wrapper for [tailwind-styled-components](https://www.npmjs.com/package/tailwind-styled-components) Exposing a small utility for nested inline intellisense and some typing fixes. |
+| [@trackunit/ui-icons](https://www.npmjs.com/package/@trackunit/ui-icons) | Used by the Icon component in [@trackunit/react-components](https://www.npmjs.com/package/@trackunit/react-components) |
+| [@trackunit/ui-design-tokens](https://www.npmjs.com/package/@trackunit/ui-design-tokens) | TODO. |
```

Comment:
Exposes the design tokens used to generate the Tailwind Config, as well as a color utility used to get colors from the theme without using css variables.